__tests__/components/AnalysisContainer/AnalysisContainer.test.tsx

```
@@ -363,4 +363,147 @@ describe("AnalysisContainer", () => {
+          CM: ["custom-cm"],
+          SKU: ["custom-sku"],
+          NVPN: ["custom-nvpn"],
+        },
+      },
+      isLoading: false,
+      isError: false,
+    };
+
+    mockUseChat.mockReturnValue(mockAnalysisWithMetadata);
+
+    let capturedInitialFilters: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)